appveyor.yml

```diff
@@ -2,4 +2,7 @@ version: 0.0.{build}
 build_script:
 - cmd: dotnet build Teq.sln
 test_script:
+- cmd: cd Test
+- cmd: nuget install Appveyor.TestLogger -Version 2.0.0
```

Comment:
More idiomatic given you're using `dotnet` elsewhere to call `dotnet add package` instead of `nuget install`.

Reply:
True, but I don't want to modify the project file itself, just grab the binaries from nuget. Is there a way to do that with the dotnet cli?

Reply:
Oh sorry, I see. Probably not, no. Looks like dotnet test isn't picking the DLL up for you - their docs suggest `--test-adapter-path:.`.